roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -119,8 +126,9 @@ projectConfig:
     mcsLabelsPerProject: {{ openshift.master.mcs_labels_per_project }}
     uidAllocatorRange: "{{ openshift.master.uid_allocator_range  }}"
 routingConfig:
-  subdomain:  "{{ openshift.master.default_subdomain | default("") }}"
+  subdomain:  "{{ openshift.master.default_subdomain | default("router.default.svc.cluster.local") }}"
```

Comment:
I'll check - these defaults are from the original PR so I assumed they were here for a reason.